Disable use of custom allocator (jemalloc not supported)
See issue#2952 https://github.com/charliermarsh/ruff/issues/2952
Index: crates/ruff_cli/src/main.rs
--- crates/ruff_cli/src/main.rs.orig
+++ crates/ruff_cli/src/main.rs
@@ -24,21 +24,6 @@ mod iterators;
 mod printer;
 mod resolve;
 
-#[cfg(target_os = "windows")]
-#[global_allocator]
-static GLOBAL: mimalloc::MiMalloc = mimalloc::MiMalloc;
-
-#[cfg(all(
-    not(target_os = "windows"),
-    any(
-        target_arch = "x86_64",
-        target_arch = "aarch64",
-        target_arch = "powerpc64"
-    )
-))]
-#[global_allocator]
-static GLOBAL: tikv_jemallocator::Jemalloc = tikv_jemallocator::Jemalloc;
-
 enum ExitStatus {
     /// Linting was successful and there were no linting errors.
     Success,
